src/junit.rs

```diff
@@ -158,11 +161,131 @@ pub fn get_position_info(input: &[u8], byte_offset: usize) -> (usize, usize) {
     (line, column)
 }
 
+#[derive(Error, Debug)]
```

Comment:
maybe you can use thiserror here:

```suggestion
#[derive(Error, Debug)]
#[error("not evals property")]
struct NotEvalsPropertyError;
```